### PR TITLE
Cleans main.rs and fixes the "graph" subcommand

### PR DESCRIPTION
**Pull request by jenrik**

### CODE REVIEW

File: src/main.rs [hunk 1]
```diff
@@ -57,120 +58,149 @@ fn main() -> Result<(), Box<dyn Error>> {
...
+            let formula = load_formula(formula_path);
 
-    let v0 = game_structure.initial_state_index();
+            handle_json(graph, formula)
+        }
+        "lcgs" => {
+            let lcgs = parse_lcgs(&content).unwrap_or_else(|err| {
+                eprintln!("Couldn't open input model\n\nError:\n{}", err);
+                exit(1);
+            });
```
Comment (NicEastvillage) [inline src/main.rs:188]: This error would be a parsing error. File IO errors are handled above.

File: src/main.rs [hunk 2]
```diff
@@ -57,120 +58,149 @@ fn main() -> Result<(), Box<dyn Error>> {
...
+            let game_structure = IntermediateLCGS::create(lcgs).unwrap_or_else(|err| {
+                eprintln!("Couldn't open input model");
+                exit(1);
+            });
```
Comment (NicEastvillage) [inline src/main.rs:192]: This error would be a semantics error. The user wrote something invalid.

**Review by NicEastvillage — CHANGES_REQUESTED** [review]
Nice! This is much better. Only 8 lines of of dublicate code.

You might now be done with this PR from what I can tell, but here are some thoughts from me:
* This PR overshadows PR #58. Can you including infering of model type in this PR please?
* I would prefer if you wrote "Could not" or even better "Failed to" instead of "Couldn't" in the error messages.
* Comments are missing

**Review by NicEastvillage — APPROVED** [review]

**Review by GeneralDenmark — APPROVED** [review]
Looking Good :tm:

**Pull request merged**